packages/core-web/src/styles/markbind.css

```diff
@@ -21,14 +21,21 @@ pre.hljs > code {
 
 pre > code.hljs {
     -webkit-background-clip: padding-box; /* for Safari */
-    background: ghostwhite;
     background-clip: padding-box; /* for IE9+, Firefox 4+, Opera, Chrome */
     border: 1px solid rgb(200, 200, 200);
     border: 1px solid rgba(200, 200, 200, 0.3);
     border-radius: 5px;
     counter-reset: line;
 }
 
+pre > code.hl-light {
+    background: ghostwhite;
+}
+
+pre > code.hl-dark {
+    background: #1e1e1e;
```

Comment:
I have no comments about the vs2015 theme choice, maybe we can get @damithc's opinion though

the background here does seem a little too dark though, propose: `#333333` (lightness +8%) if its not just me =P

Reply:
the line number needs and heading background to be brightened as well
https://deploy-preview-1355--markbind-master.netlify.app/userguide/formattingcontents#syntax-coloring
https://deploy-preview-1355--markbind-master.netlify.app/userguide/formattingcontents#heading

Reply:
Ah the background is actually identical to the default theme in VS Code so personally I've grown accustomed to it haha, the line numbering color is also similar. But `#333333` is also not bad as an alternative. Would like to know what @damithc prefers as well

Reply:
What are my choices? I don't mind going with a scheme used by another popular tool.

Reply:
For heading background, do we need to match it with the code theme? The styling for heading hasn't been affected so far (it's still the same as master), but if we're to match it with the theme, could be a bit tricky as well as I'm planning to target `'pre'` nodes in `ComponentParser` to apply the theme but the heading is outside that.

Reply:
@damithc The theme choices are available on Highlight.js' demo here https://highlightjs.org/static/demo/ (there are loads of it, light and dark). The dark theme implemented in this PR is `Vs 2015`, but it's just my pick as it is very similar to VS Code which I'm very familiar with. You can check how it looks like on the links @ang-zeyu posted above.

Also, if you're okay with it, are you also okay with the current background color (re: `#1e1e1e` or `#333333`)?

Reply:
Let's go with VS-like theme. Re background color, hard to decide without a side-by-side comparison in-action.

Reply:
@damithc Okay :+1: Here's the comparison for the background colors (left `#1e1e1e`, right `#333333`):

![image](https://user-images.githubusercontent.com/24468110/93791810-481e5580-fc67-11ea-90c6-c87b3e38e9de.png)

Reply:
PyCharm (seems to be in the middle):
![image](https://user-images.githubusercontent.com/1673303/93792378-0b069300-fc68-11ea-855e-cd08df9cc53a.png)

So, something in the middle? But frankly, I have no strong objections to either of the three.

Reply:
Oh yes, JetBrains IDEs use that background color. I think I'm going to use that instead, it's a good middle ground. Thanks!

@ang-zeyu for the line numbering color, I think I'll match it with the normal text color on the code block. For heading background, I'm not sure whether I should change it. The styling's still untouched from master, and IMO the current one is still okay-ish for dark, it kind of contrasts with the code block well (?). What's your opinion on this?

Reply:
>  for the line numbering color, I think I'll match it with the normal text color on the code block.

👍 

> For heading background, I'm not sure whether I should change it.

looks a little odd 👀 

![Untitled](https://user-images.githubusercontent.com/3306138/93872863-591ca480-fd03-11ea-8e23-e937ff2813f7.png)

Reply:
> for the line numbering color, I think I'll match it with the normal text color on the code block.

Perhaps better for the line numbers to appear less prominent than normal text? They are two different things and the line numbers are less important, right?

Reply:
> Perhaps better for the line numbers to appear less prominent than normal text? They are two different things and the line numbers are less important, right?

Hmm, now that you mention it, editors do stylize the line numbers to be a bit darker than normal text. Just found out that the color that we're using regularly (`#888888`) is very similar, almost identical to what VS Code uses (`#858585`). JetBrains IDEs uses approximately `#606366`, which is even darker than the previous two.

Personally, I don't see any problem with the current color, but I can also see if you guys think it should be brightened to make the line numbers more stand out. Any opinions? @ang-zeyu

Reply:
It's less about the absolute color but more about how line numbers appear relative to normal code.

Reply:
Sorry, missed the ping!

> Perhaps better for the line numbers to appear less prominent than normal text? They are two different things and the line numbers are less important, right?

> Personally, I don't see any problem with the current color, but I can also see if you guys think it should be brightened to make the line numbers more stand out. Any opinions?

ok, good point. let's keep the current line color then

Reply:
> but I can also see if you guys think it should be brightened to make the line numbers more stand out.

I was suggesting line numbers should stand out _less_, not more, as they are less important than the code.

Reply:
> ![Untitled](https://user-images.githubusercontent.com/3306138/93872863-591ca480-fd03-11ea-8e23-e937ff2813f7.png)

Like this image? (current) It is already slighty darker than the code text

Reply:
> Like this image? (current) It is already slighty darker than the code text

Yes, that works. I thought the same color is being used (I saw something like that earlier in the thread).

Reply:
ahhh no that was my original suggestion to @ryoarmanda 

> Hmm, now that you mention it, editors do stylize the line numbers to be a bit darker than normal text.

lets go with this instead, as you guys mentioned the line numbers shouldn't receive too much attention

Reply:
Okay then, we'll use `#333333` for the line numbers, thanks all :+1:

Reply:
> Edit: oh wait, that is the original color! It's the same as the light style 😅

yes, we went full circle 😅